File: Unidad3/Java/u3p2.java
package com.mycompany.Programa2_U3;

import java.util.Scanner;

public class u3p2 {

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int C1,C2,C3,C4,C5,C6,prom;
        System.out.println("Ingresa la 1ra calificacion ");
        C1 =sc.nextInt();
        System.out.println("Ingresa la 2da calificacion ");
        C2 =sc.nextInt();
        System.out.println("Ingresa la 3ra calificacion ");
        C3 =sc.nextInt();
        System.out.println("Ingresa la 4ta calificacion ");
        C4 =sc.nextInt();
        System.out.println("Ingresa la 5ta calificacion ");
        C5 =sc.nextInt();
        System.out.println("Ingresa la 6ta calificacion ");
        C6 =sc.nextInt();
        
        prom=(C1+C2+C3+C4+C5+C6)/6;
        
        if (prom>=70){
        System.out.println("Aprobado"); 
        }
        else{
        System.out.println("Reprobado");
        }
    }
}

File: Unidad3/C#/u3p2.cs
using System;

namespace u3p2
{
    internal class Program
    {
        static void Main(string[] args)
        {

            int C1, C2, C3, C4, C5, C6, promedio;
            Console.WriteLine("Ingresa la 1ra calificación ");
            C1 = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("ingresa la 2da calificación ");
            C2 = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Ingresa la 3ra calificación ");
            C3 = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Ingresa la 4ta calificación ");
            C4 = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Ingresa la 5ta calificación ");
            C5 = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Ingresa la 6ta calificación ");
            C6 = Convert.ToInt32(Console.ReadLine());

            promedio = (C1 + C2 + C3 + C4 + C5 + C6) / 6;
            Console.WriteLine("Tu calificacion es de: " + promedio);
            if (promedio >= 70)
            {
                Console.WriteLine("Aprobado");
            }
            else
            {
                Console.WriteLine("Reprobado");
            }
        }
    }
}